src/containers/PrayerRequest/PrayerRequest.css

```diff
@@ -0,0 +1,15 @@
+#prayer-request {
```

Comment:
css please remove

Reply:
Decided to use CSS for now. SASS removed in latest push